vm/systemSmartContracts/esdt_test.go

```diff
@@ -157,11 +157,11 @@ func TestEsdt_ExecuteIssue(t *testing.T) {
 	output = e.Execute(vmInput)
 	assert.Equal(t, vmcommon.FunctionWrongSignature, output)
 
-	vmInput.Arguments[0] = []byte("01234567891")
+	vmInput.Arguments = append(vmInput.Arguments, []byte("ticker"))
```

Comment:
the third argument is not initialSupply? Is it ok with "ticker"?

Reply:
done

Reply:
ok